docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -118,6 +104,23 @@ Can only be called by the current owner.
 
 Emits an xref:Ownable-OwnershipTransferred[OwnershipTransferred] event.
 
+[.contract-item]
+[[Ownable-initializer]]
+==== `[.contract-item-name]#++initializer++#++(ref self: ContractState, owner: ContractAddress)++` [.item-kind]#internal#
```

Comment:
we should add `External` and `Internal` titles here to for consistency with ERC20